meta-mender-core/recipes-update-modules/mender-app-update-module/mender-app-update-module_git.bb

```
@@ -0,0 +1,68 @@
+DESCRIPTION = "Mender Application Update Module for supporting containerized application updates on devices including related tools."
+
+HOMEPAGE = "https://mender.io"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=b4b4cfdaea6d61aa5793b92efd42e081"
+
+SRC_URI = "git://github.com/mendersoftware/app-update-module;branch=master;protocol=https"
+
+SRCREV = "15025d9f1e21eafcbf1325b2a5034b2606e1d719"
+
+S = "${WORKDIR}/git"
+
+RDEPENDS:${PN} = "jq mender-client xdelta3"
+
+PACKAGECONFIG ?= ""
+PACKAGECONFIG[docker-compose] = ",,,docker-compose"
+PACKAGECONFIG[k3s] = ",,,packagegroup-k3s-node"
+PACKAGECONFIG[k8s] = ",,,kubectl"
+
+do_compile() {
+    if ! ${@bb.utils.contains('PACKAGECONFIG', 'docker-compose', 'true', 'false', d)} && \
+       ! ${@bb.utils.contains('PACKAGECONFIG', 'k3s', 'true', 'false', d)} && \
+       ! ${@bb.utils.contains('PACKAGECONFIG', 'k8s', 'true', 'false', d)}; then
+        bbwarn "PACKAGECONFIG for mender-app-update-module is empty or invalid. Please specify either `docker-compose`, `k3s` or `k8s` unless you know what you are doing."
```

```suggestion
        bbwarn "PACKAGECONFIG for mender-app-update-module is empty or invalid. Please specify either 'docker-compose', 'k3s' or 'k8s' unless you know what you are doing."
```

It won't print the quoted things otherwise:

```
WARNING: mender-app-update-module-git-r0 do_compile: PACKAGECONFIG for mender-app-update-module is em
pty or invalid. Please specify either ,  or  unless you know what you are doing.
```